packages/compass-aggregations/src/stores/store.ts

```diff
@@ -237,6 +229,12 @@ export function activateAggregationsPlugin(
 
   store.dispatch(updatePipelinePreview());
 
+  store.dispatch(
+    maxTimeMSChanged(
+      preferences.getPreferences().maxTimeMS ?? DEFAULT_MAX_TIME_MS
+    )
+  );
```

Comment:
In pipeline-collation.tsx we were so far differentiating the cases of "user has not set a maxTimeMS", "user inherits maxTimeMS from preferences/default" and "user has explicitly set a maxTimeMS for this aggregation"

That goes away if we approach it this way, I'd maybe figure out some way to keep the current behavior, but if we can't, then we should probably also refactor the pipeline-options.tsx component